src/test/java/LottoMachineTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoMachineTest {
```

Comment:
아래와 같이 클래스에도 `@DisplayName` 을 사용할 수 있어요~
```suggestion
@DisplayName("LottoMachine 테스트")
class LottoMachineTest {
```

Reply:
클래스에서도 @DisplayName을 설정할 수 있군요! 처음 알게 된 사실이네요. 잘 활용해보겠습니다!